src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminal.accessibility.contribution.ts

```diff
@@ -40,7 +40,7 @@ class TextAreaSyncContribution extends DisposableStore implements ITerminalContr
 		super();
 	}
 	xtermReady(xterm: IXtermTerminal & { raw: Terminal }): void {
-		const addon = this._instantiationService.createInstance(TextAreaSyncAddon, this._instance.capabilities);
+		const addon = this._instantiationService.createInstance(TextAreaSyncAddon, this._instance.capabilities, this._instance.onDidFocus);
```

Comment:
Instead of passing the event around you can just hook up a standard `focus` DOM listener to `xterm.textarea`, there's the add disposable dom listener helper to help

Reply:
that's what I tried initially, and it didn't work. I'll try again.

Reply:
wasn't working for the same reason that it's still not working, so better to go with this instead